notebooks/book2/26/gan_mixture_of_gaussians.ipynb

```diff
@@ -0,0 +1,818 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %pip install -qq git+https://github.com/google/flax.git
</pre><p>Remove this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>